lib/lint.ml

```diff
@@ -21,11 +20,23 @@ module Make (Docker : S.DOCKER_CONTEXT) = struct
     @@ install_ocamlformat
     @@ copy ~chown:"opam" ~src:["./"] ~dst:"./" ()
 
-  let v_fmt ~ocamlformat_source ~base ~src =
+  let ocamlformat_run ~ocamlformat_source ~base ~src =
```

Comment:
We discussed this before: this _isn't_ just an OCamlformat check! `run_fmt` or similar will do.

Reply:
It is ! And I wanted to make that obvious (it takes `ocamlformat_source and calls `ocamlformat_dockerfile`).
I agree we should split concern so I moved the "run" part in an other function.

Reply:
We'll have to agree to disagree. Your separation works, anyway.

Reply:
Note that this function is not exposed.